README.md

```diff
@@ -49,11 +49,11 @@ go get github.com/onflow/flow-go-sdk
 
 ### Generating Keys
 
-Flow uses [ECDSA key pairs](https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm) 
+Flow uses [ECDSA](https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm) 
```

Comment:
`ECDSA`, and not the key pairs only, includes all algorithms of the scheme, which are all needed by Flow.